ext/gd/tests/bug38212-mb.phpt

```diff
@@ -2,6 +2,12 @@
 Bug #38212 (Seg Fault on invalid imagecreatefromgd2part() parameters)
 --EXTENSIONS--
 gd
+--SKIPIF--
+<?php
+    if (!GD_BUNDLED && version_compare(GD_VERSION, '2.3.3', '>=')) {
+        die("skip test requires GD 2.3.2 or older");
```

Comment:
```suggestion
        die("skip test requires a version older than GD 2.3.3");
```
As the current wording is kinda weird IMHO

Reply:
If it's easy for you to do now, that would be convenient.

Reply:
On second thought, it looks like I chose this wording to match the existing version-based "skip" messages. For example, https://github.com/php/php-src/blob/master/ext/gd/tests/bug43121.phpt#L8

That same phrase (with various version triplets) already appears 25 times in `ext/gd/tests`. I'm still happy to change it though if this doesn't change anyone's mind.

Reply:
I see. IMO if it's already this way anyway, it can be kept. It's not that important after all.
If we end up disliking the wording I can simply amend it later.

Reply:
Right, if it's already liek this leave it then